features/promotion/managing_catalog_promotions/editing_catalog_promotion.feature

```diff
@@ -46,12 +48,19 @@ Feature: Editing catalog promotion
         Then I should not be able to edit its code
 
     @api @ui @javascript
-    Scenario: Editing catalog promotion rule
+    Scenario: Editing catalog promotion variant rule
         When I edit "Christmas sale" catalog promotion to be applied on "Kotlin T-shirt" variant
         Then I should be notified that it has been successfully edited
         And this catalog promotion should be applied on "Kotlin T-shirt" variant
         And this catalog promotion should not be applied on "PHP T-Shirt" variant
 
+    @api @ui @javascript
+    Scenario: Editing catalog promotion taxon rule
+        When I edit "Christmas sale" catalog promotion to be applied on "Clothes" taxon
+        Then I should be notified that it has been successfully edited
+        And this catalog promotion should be applied on "Kotlin T-shirt" variant
+        And this catalog promotion should not be applied on "PHP T-Shirt" variant
```

Comment:
`And this catalog promotion should be applied on "Clothes" taxon` it's important 😄